lib/bb/client.rb

```diff
@@ -9,13 +9,15 @@
 
 module BB
   # Core class responsible for api interface operations
+  class BBServiceException < Common::Exceptions::BackendServiceException; end
   class Client < Common::Client::Base
     include Common::Client::MHVSessionBasedClient
     include Common::Client::StreamingClient
     include SentryLogging
 
     configuration BB::Configuration
     client_session Rx::ClientSession
+    use_service_exception BBServiceException
```

Comment:
Since Common::Configuration already defines a mnemonic for the service name, i think you could possible remove the need for "use_service_exception" and just define service_exception on Common::Configuration, it can be automatically generated using minimal meta programming and will help to ensure some consistency.